src/locales/en-US.json

```diff
@@ -1701,12 +1701,66 @@
       }
     },
     "modal": {
+      "alert": {
+        "available": "Network settings is available only for Non HMC managed systems, is in PHYP standby and Default partition environment is IBM i or Default."
+      },
       "confirmRebootMessage": "Are you sure you want to reboot?",
       "confirmRebootMessage2": "A system dump is being offloaded. Rebooting will result in the dump being lost.",
       "confirmRebootTitle": "Server reboot will cause outage",
       "confirmShutdownMessage": "Are you sure you want to shut down?",
       "confirmShutdownMessage2": "A system dump is being offloaded. Shutting down the system will result in the dump being lost.",
-      "confirmShutdownTitle": "Server shutdown will cause outage"
+      "confirmShutdownTitle": "Server shutdown will cause outage",
+      "networkSettings": {
+        "advanced": "Advanced",
+        "advancedSettings": {
+          "vlanTagId": "VLAN Tag Identifier",
+          "maxFrameSize": "Maximum Frame Size"
+        },
+        "cancel": "Cancel",
+        "header": "Network settings",
+        "ipAddressProtocolHeading": "IP Address Protocol",
+        "networkTypeHeading": "Network Install Type",
+        "partition": "Partition",
+        "partitionSettings": {
+          "ipAddress": "IP Address",
+          "subnetMask": "Subnet Mask",
+          "gateway": "Gateway"
+        },
+        "server": "Server",
+        "serverSettings": {
+          "bootServerIPAddress": "Boot Server IP Address",
+          "imageDirectoryPath": "Image Directory Path",
+          "targetIpAddress": "Target IP Address",
+          "targetName": "Target Name",
+          "initiatorName": "Initiator Name",
+          "targetPort": "Target Port",
+          "chapName": "Chap Name",
+          "chapSecret": "Chap Secret",
+          "restoreDefault": "Restore default"
+        },
+        "settingsDescription": "Configure the network boot settings for the IBM i partition.",
+        "settingsHeading": "Configure Network settings",
+        "submit": "Submit",
+        "toast": {
+          "errorSavedSettings": "Error saving Network settings",
+          "errorRestoreDefault": "Error restoring default initiator name",
+          "errorUpdateDMode": "Error updating the IBM i boot partition mode",
+          "successRestoreDefault": "Successfully restored default initiator name",
+          "successSavedSetting": "Successfully saved Network boot settings for IBMi. Please press the `Continue to OS running` link.",
```

Comment:
Rename as "Successfully saved network boot settings for IBMi. Please press the `Continue to OS running` link."

Reply:
I have made change in the error message to keep this consistent. Error saving Network settings. so, this "Network" would match each other.